index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="pages/styles/style.css" rel="stylesheet">
+
+</head>
+<body>
+  <h1> Roza Ajema's Portfolio</h1>
+  <nav class = "nav">
+<ul>
+  <li><a href="index.html">Home</a> </li>
+  <li><a href="pages/about.html">About</a> </li>
+  <li><a href="pages/portfolio.html">Portfolio</a></li>
+</ul>
+  </nav>
+<div class="wrapper">
```

Comment:
Instead of making this a `<div>` with the class `wrapper` it would probably have been more appropriate to use `<main>`:

```suggestion
<main>
```